pysr/sr.py

```diff
@@ -139,11 +141,11 @@ def _maybe_create_inline_operators(binary_operators, unary_operators):
 
 
 def _check_assertions(
-    X,
-    use_custom_variable_names,
-    variable_names,
-    weights,
-    y,
+        X,
```

Comment:
Are these formatting changes made by your IDE? Most of these changes don't look necessary for the PR.

Reply:
They were, I have undone those now, sorry!